BAU: Fix docker compose build/context paths to align them with latest…

undermad opened the pull request:
… folders structure generated by dev-deploy tool.

## Proposed changes
### What changed

- Changed build/context paths to Dockerfiles 

### Why did it change

- To align them with project folder structure generated by dev-deploy tool

### Issue tracking
<!-- Jira ticket & other docs, like RFCs -->

- BAU

## Checklists

- [x] READMEs and documentation up-to-date
- [ ] API/ unit/ contract tests have been written/ updated
- [ ] No risk of exposure: PII, credentials, etc through logs/ code
- [ ] Production changes appropriately staged out

undermad committed:
BAU: Fix docker compose build/context paths to align them with latest folders structure generated by dev-deploy tool.

undermad committed:
BAU: Update README.md - Explain the paths to Dockerfiles for compose.yaml file.

thebauSoftwire reviewed (dismissed):
nice! 🎉

thebauSoftwire commented on local-running/compose.yaml.template:
I wonder if we should create a template for this for devs to copy over? Then add this to the `.gitignore`. If someone was to update this, they might accidentally commit it 🤔 To allow devs to have their own setup... New devs will likely have the dev-deploy but a mix of devs have the folders symlinked to other places

undermad replied:
Agree, this is reasonable. Updated the PR according to your suggestion.

undermad committed:
BAU: Change compose.yaml to be a template and add local-running/compose.yaml to .gitignore

As folder structure may be different for developers, especially before dev-deploy tool, changing compose.yaml to a template will prevent accidental commits if someone will use different path to the Dockerfiles
